### PR TITLE
docs: app upgrade notes for Substra 0.23.0

### PR DESCRIPTION
**Pull request by guilhem-barthes**
Signed-off-by: Guilhem Barthes <guilhem.barthes@owkin.com>

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,6 +7,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+- Add upgrade note for 0.23.0 (#228)
```
Comment (ThibaultFy) [inline CHANGELOG.md:10]:
```suggestion
- Add upgrade note for 0.23.1 (#228)

```

File: docs/source/operations/upgrade_notes.rst [hunk 2]
```diff
@@ -0,0 +1,13 @@
+Upgrade notes
+=============
+
+Substra 0.23.0
```
Comment (ThibaultFy) [inline docs/source/operations/upgrade_notes.rst:4]:
```suggestion
Substra 0.23.1
```

**Comment by ThibaultFy** [comment]
I don't know why but we were not automatically set à reviewers on this one (did not receive any notification)

File: docs/source/operations/upgrade_notes.rst [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+--------------
+
+This version ships Redis *with persistence (AOF) activated*. As this component is used as a message broker and not as a cache, the previous redis deployment can be removed before lauching the upgrade.
+
+For each cluster where substra-backend is deployed, run the following command *before upgrading*:
```
Comment (oleobal) [inline docs/source/operations/upgrade_notes.rst:9]: Perhaps also mention that no tasks must be running

File: docs/source/operations/upgrade_notes.rst (hunk 2 above)
Comment (ThibaultFy) [reply docs/source/operations/upgrade_notes.rst:4]: @guilhem-barthes up

**Comment by ThibaultFy** [comment]
Don't hesitate to re-ask for review (re-request review button) when you update the PR :)

**Review by ThibaultFy — APPROVED** [review]
Thanks :)

**Pull request merged**